kotlinpoet/build.gradle.kts

```diff
@@ -23,11 +24,12 @@ spotless {
   kotlin {
     targetExclude(
       // Non-Square licensed files
-      "src/commonMain/kotlin/com/squareup/kotlinpoet/ClassName.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/AbstractTypesTest.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/ClassNameTest.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/TypesEclipseTest.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/TypesTest.kt",
+      "src/*Main/kotlin/com/squareup/kotlinpoet/ClassName.kt",
+      "src/*Main/kotlin/com/squareup/kotlinpoet/ClassName.*.kt",
```

Comment:
What extensions does this pattern target?

Reply:
Since the original `ClassName.kt` was migrated to commonMain, jvmMain, jsMain, wasmJsMain, etc., this pattern corresponds to the `ClassName` file under each platform module.
The other `*Test` actually correspond to files that currently exist only in `jvmTest`.